app/workers/import_worker.rb

```diff
@@ -0,0 +1,15 @@
+class ImportWorker
+  include Sidekiq::Worker
+
+  def perform(device, json_file)
```

Comment:
@dmitrytrager Perform takes about 15 minutes to finish when you are uploading 30k check-ins. I can't figure out how to speed it up as you have to pass the csv file contents as an JSON type object to the ImportWorker. Any ideas? Think you will have basically written this whole branch by the time we're through haha...